test/evergreen.yml

```diff
@@ -28,6 +29,39 @@ exec_timeout_secs: 21600 # 6 hrs
 #######################################
 
 functions:
+  "setup environment":
+    - command: expansions.update
+      params:
+        updates:
+        # The expansion is used for each task that runs a WiredTiger test. All of these variables
+        # are common among the build variants, if there are any specific variables that needs to be
+        # set, users can add onto the additional_env_vars in the variant
+        - key: PREPARE_TEST_ENV
+          value: |
+            export WT_TOPDIR=$(git rev-parse --show-toplevel)
+            export WT_BUILDDIR=$WT_TOPDIR/cmake_build
+
+            if [ "$OS" = "Windows_NT" ]; then
+              export PATH=/cygdrive/c/python/Python311:/cygdrive/c/python/Python311/Scripts:$PATH
+              export PYTHONPATH="($WT_TOPDIR/lang/python/wiredtiger):$(cygpath -w $WT_TOPDIR/lang/python)"
+            else
+              export PATH=/opt/mongodbtoolchain/v4/bin:$PATH
+              export LD_LIBRARY_PATH=$WT_BUILDDIR
+            fi
+
+            if [[ "${CMAKE_PREFIX_PATH|}" =~ TCMALLOC_LIB ]]; then
+              export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$WT_TOPDIR/TCMALLOC_LIB/lib"
+              export LD_PRELOAD=$WT_TOPDIR/TCMALLOC_LIB/lib/libtcmalloc.so
+            fi
+            ${additional_env_vars}
```

Comment:
There seems to be inconsistency in the later setting points - some have `export` (making the expansion a command) others do not.

Reply:
fixed! [3856cab](https://github.com/wiredtiger/wiredtiger/pull/9533/commits/3856cab1ed5f29fae68eca15064a0b18d9609cfb)